src/frontend/src/components/FillForms/UserDataForm.tsx

```diff
@@ -177,9 +295,32 @@ class UserDataForm extends React.Component<Props, State> {
                 />
               </div>
               <div className="mb4">
-                <label htmlFor="phoneNumber" className="db mb1 fw6">
-                  Phone Number
-                </label>
+                <div className="flex">
+                  <label htmlFor="phoneNumber" className="db mb1">
+                    <span className="fw6"> Phone Number </span>{" "}
+                    <span className="fw2">xxx-xxx-xxxx </span>
+                  </label>
+
+                  <input
+                    id="isInternationalNumber"
```

Comment:
I feel like this part is unnecessary. I personally never put in hyphens for online forms. Maybe just check for at least one number?

Reply:
Agreed, in the past I have just checked for at least 7 digits or something to make sure int is covered. I think it gets to a point where validation isn't helping much for phones since there are so many possibilities and valid ways of inputting.